src/lib.rs

```diff
@@ -268,6 +276,25 @@ impl ProgLog {
         }
     }
 
+    /// Increment the progress logger by `count` and check if a new message should be emitted.
```

Comment:
suggestion: I think this (and the other _count method) should say something about how this doesn't carry over the count for the emission test. this will only emit `count // unit` times, or something like that.

Reply:
I opted for stating this is equivalent to calling `record` `count` number of times.